packages/runtime/src/errorHandler.ts

```diff
@@ -2,7 +2,7 @@ import {
   ClientComputationFunction,
   Dependency,
   ServerComputationFunction,
-  SSRComputationFunction,
+  SSRComputationHook,
```

Comment:
Renamed to differentiate between the `SSR Computation Function` that's the function written by user and get executed when needed. And the `SSR Computation hook` which is the type of the hooks being called by the plugin which are `useSSRComputation_Client` and `useSSRComputation_Server`